src/containers/tutor-home-page/subjects-step/SubjectsStep.jsx

```diff
@@ -1,13 +1,228 @@
+  ]
+  const [selectedCategory, setSelectedCategory] = useState('')
+  const [selectedSubject, setSelectedSubject] = useState('')
+  const [addedSubjects, setAddedSubjects] = useState([])
+
+  useEffect(() => {
+    setSelectedSubject('')
+  }, [selectedCategory])
+
   return (
-    <Box sx={styles.container}>
-      <Box sx={styles.rigthBox}>
-        Subjects step
-        {btnsBox}
+    <Box sx={{ ...styles.container, alignItems: 'flex-start' }}>
+      <Box>
+        <img
+          alt='Study Category'
+          src={studyCategoryImg}
+          style={{ maxWidth: 320, marginBottom: 24 }}
+        />
+      </Box>
+      <Box
+        sx={{
+          ...styles.rigthBox,
+          maxWidth: 380,
+          flex: 1,
+          display: 'flex',
+          flexDirection: 'column',
+          minHeight: 520,
+          marginLeft: 0
+        }}
+      >
+        <div style={{ flex: 1, display: 'flex', flexDirection: 'column' }}>
+          <div
+            style={{
+              marginBottom: 16,
+              color: 'black',
+              fontSize: 16,
+              fontWeight: 300
+            }}
+          >
+            Velit officia consequat duis enim velit mollit. Other categories you
+            can add in your account settings later.
+          </div>
+          <FormControl sx={{ width: '100%', marginBottom: 3 }}>
+            <InputLabel id='category-label'>Main Tutoring Category</InputLabel>
+            <Select
+              label='Main Tutoring Category'
+              labelId='category-label'
+              onChange={(e) => {
+                setSelectedCategory(e.target.value)
+                setAddedSubjects([])
+                setSelectedSubject('')
+                handleStepData('subjects', [])
+              }}
+              value={selectedCategory}
+            >
+              {categories.map((cat) => (
+                <MenuItem
+                  key={cat._id || cat.id || cat.name}
+                  value={cat._id || cat.id || cat.name}
+                >
+                  {cat.name}
+                </MenuItem>
+              ))}
+            </Select>
+          </FormControl>
+          <FormControl sx={{ width: '100%', marginBottom: 3 }}>
+            <InputLabel id='subject-label'>Subject</InputLabel>
+            <Select
+              label='Subject'
+              labelId='subject-label'
+              onChange={(e) => {
+                setSelectedSubject(e.target.value)
+              }}
+              value={selectedSubject}
+            >
+              {subjects
+                .filter((sub) => sub.categoryId === selectedCategory)
+                .map((sub) => (
+                  <MenuItem
+                    key={sub._id || sub.id || sub.name}
+                    value={sub._id || sub.id || sub.name}
+                  >
+                    {sub.name}
+                  </MenuItem>
+                ))}
+            </Select>
+          </FormControl>
+          <button
```

Comment:
```suggestion
import Button from '@mui/material/Button'
          <Button
```
it would be better maybe to use button from MUI as in the whole project